electron/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "abphelper-gui",
   "productName": "AbpHelper",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
More changes should do to change the version (such as changing the version in common.props and adding a new Git tag). I will change the version myself after this PR merge.